plugins/become/ksusu.py

```diff
@@ -0,0 +1,182 @@
+              - name: ANSIBLE_KSUSU_USER_SU
+            required: False
+        become_exe_ksu:
+            description: Ksu executable
+            default: ksu
+            ini:
+              - section: privilege_escalation
+                key: become_exe_ksu
+              - section: ksusu_become_plugin
+                key: executable_ksu
+            vars:
+              - name: ansible_become_exe_ksu
+              - name: ansible_ksusu_exe_ksu
+            env:
+              - name: ANSIBLE_BECOME_EXE_KSU
+              - name: ANSIBLE_KSU_EXE_KSU
+        become_exe_su:
+            description: Absolute path to the su executable
+            default: /bin/su
+            ini:
+              - section: privilege_escalation
+                key: become_exe_su
+              - section: ksusu_become_plugin
+                key: executable_su
+            vars:
+              - name: ansible_become_exe_su
+              - name: ansible_ksusu_exe_su
+            env:
+              - name: ANSIBLE_BECOME_EXE
+              - name: ANSIBLE_BECOME_EXE_SU
+              - name: ANSIBLE_KSUSU_EXE_SU
+        become_flags_ksu:
+            description: Options to pass to ksu
+            default: '-Z -q'
+            ini:
+              - section: privilege_escalation
+                key: become_flags_ksu
+              - section: ksusu_become_plugin
+                key: flags_ksu
+            vars:
+              - name: ansible_become_flags_ksu
+              - name: ansible_ksusu_flags_ksu
+            env:
+              - name: ANSIBLE_BECOME_FLAGS_KSU
+              - name: ANSIBLE_KSU_FLAGS_KSU
+        become_flags:
+            description: Options to pass to su
+            default: ''
+            ini:
+              - section: privilege_escalation
+                key: become_flags
+              - section: privilege_escalation
+                key: become_flags_su
+              - section: ksusu_become_plugin
+                key: flags_su
+            vars:
+              - name: ansible_become_flags
+              - name: ansible_become_flags_su
+              - name: ansible_ksusu_flags
+              - name: ansible_ksusu_flags_su
+            env:
+              - name: ANSIBLE_BECOME_FLAGS
+              - name: ANSIBLE_BECOME_FLAGS_SU
+              - name: ANSIBLE_KSU_FLAGS_SU
+        become_pass:
+            description: ksu password
+            required: False
+            vars:
+              - name: ansible_ksusu_pass
+              - name: ansible_become_pass
+              - name: ansible_become_password
+            env:
+              - name: ANSIBLE_BECOME_PASS
+              - name: ANSIBLE_KSUSU_PASS
+            ini:
+              - section: ksusu_become_plugin
+                key: password
+        prompt_l10n:
+            description:
+                - List of localized strings to match for prompt detection
+                - If empty we'll use the built in one
```

Comment:
```suggestion
                - If empty we will use the built in one.
```
Also, why not have the default as a default for this option?

Reply:
This is used to detect the ksu password prompt. The documentation and the method that uses it are copied verbatim from the ksu become plugin (ksu.py, more than happy to add credit for that author to this plugin but I'm not sure how to do it properly in the docs).

I'm happy to change this to whatever you think would be best but if we do change it, we should change ksu.py as well;)

Reply:
In that case I would really put the value as a default here instead of listing the default in the code.

> I'm happy to change this to whatever you think would be best but if we do change it, we should change ksu.py as well;)

That can happen in a follow-up PR if you want.